### DISCUSSION TITLE
Automatically add @throws to docblock #2824

### DISCUSSION THREAD

**Discussion by WeedTest** [rectorphp/rector#8689] [Q&A] [answered]
How can i use this feature ?
I can’t find the rules 
https://github.com/rectorphp/rector/issues/2824

**Comment by TomasVotruba** [comment, marked as answer]
Hi, we removed this feature as it depended on nested calls and output got quite complex. Everyone had different preference :)

You can make a simple a custom rule to handle this to meet your needs :+1:

**Reply by WeedTest** [reply]
Thank you for your response.
